web/src/app/alerts/CreateAlertDialog/StepContent/step1.js

```diff
@@ -0,0 +1,196 @@
+import React, { useRef } from 'react'
+import {
+  Grid,
+  TextField,
+  InputAdornment,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  Paper,
+  Typography,
+  InputLabel,
+  Chip,
+} from '@material-ui/core'
+import { makeStyles, emphasize } from '@material-ui/core/styles'
+import { FormField } from '../../../forms'
+import ServiceLabelFilterContainer from '../../../services/ServiceLabelFilterContainer'
+import { Search as SearchIcon } from '@material-ui/icons'
+import FavoriteIcon from '@material-ui/icons/Star'
+import { ServiceChip } from '../../../util/Chips'
+import AddIcon from '@material-ui/icons/Add'
+
+const useStyles = makeStyles(theme => ({
+  addAll: {
+    backgroundColor: theme.palette.grey[100],
```

Comment:
The background color contrast ratio of the Add All button although it looks great, doesn't meet WCAG 2.1 requirements, which states that a contrast ratio of at least 3:1 must be met for graphical components.

We could probably get away with making it a `primary['500']` or as a prop, `<Chip color='primary' />` (same as the filter icon), with white text.

Current contrast ratio:
![image](https://user-images.githubusercontent.com/11381794/68315869-149e0100-007e-11ea-8537-644cc81b4c24.png)

---

Ratio of grey button with white text (same ratio for grey button on white background):
![image](https://user-images.githubusercontent.com/11381794/68315939-34352980-007e-11ea-9e7e-5cef9d9413d3.png)